Server/routes/route.js

```diff
@@ -1,46 +1,42 @@
 import express from 'express';
-import parcelController from '../controllers/parcelcontroller';
-import usersControllers from '../controllers/usersController';
+import parcelController from '../controllers/ParcelController';
+import usersControllers from '../controllers/UsersController';
 import middlewares from '../middlewares/index';
 
-const { validateParcels, validateRegister, validateLogin, validateToken, validateSuperAdmin, validateAdmin } = middlewares;
+const { validateParcels, validateRegister, validateLogin, validateToken, validateSuperAdmin, validateAdmin, validateLocationUpdate } = middlewares;
+
+const { updateParcelDestination, cancelParcelOrder, registerUser, login, createAdmin  } = usersControllers;
+
+const { createNewParcel, getParcelsByUser, getAllParcels, getAllParcelsBySpecificUser, updateParcelStatus, updateParcelPresentLocation } = parcelController;
 
 const router = express.Router();
 
 // this is for the home route
 router.get('/', (req, res) => {
   res.status(200).send('welcome to SendIt');
 });
-
 // this is the route for creating parcels
-// CREATE PARCELS
-router.post('/parcels', validateParcels, validateToken, parcelController.createNewParcel);
+router.post('/parcels', validateParcels, validateToken, createNewParcel);
 // this is the route to get all parcels for a user
-// GET ALL PARCELS
-router.get('/parcels/user', parcelController.getParcelsByUser);
-// this is the route to get a specific parcel
-// GET A SPECIFIC PARCEL
-// router.get('/parcels/:id', parcelController.getSpecificParcel);
-// this is to change the dropofflocation of a parcel
-// PUT IN A NEW DROPOFFLOCATION
-router.put('/parcels/:pid/newlocation', validateToken, validateAdmin, parcelController.updateParcelPresentLocation);
+router.get('/parcels',validateToken, getParcelsByUser);
+// this is to enable a user change the dropoff location of a parcel
+router.patch('/parcels/:pid/destination',validateLocationUpdate, validateToken, updateParcelDestination);
 // this is to enable a user to cancel a parcel delivery order
-router.put('/parcels/:pid/cancel', validateToken, usersControllers.cancelParcelOrder);
-// this is to delete a specific parcel order
-// DELETE A PARCEL ORDER
-// router.delete('/parcels/:id', parcelController.deleteSpecificParcel);
-
+router.patch('/parcels/:pid/cancel', validateToken, cancelParcelOrder);
```

Comment:
Np part of the rubrics instructed you to use `PATCH` request. It's rather unfortunate you don't pay attention to very simple details.

Reply:
it has been fixed